clang/test/CodeGenSYCL/stall_enable_host.cpp

```diff
@@ -2,6 +2,8 @@
 
 // Tests for IR of Intel FPGA [[intel::use_stall_enable_clusters]] function attribute on Host (no-op in IR-CodeGen for host-mode).
 
+class kernel_name_1;
+
```

Comment:
same here

Reply:
Yeah, same reason I described in previous comment. RT tests are changed for the same reason.